test/functional/legacy/memory_usage_spec.lua

```diff
@@ -143,16 +147,20 @@ describe('memory usage', function()
         call s:f()
       endfor
     ]])
+    wait()
     local after = monitor_memory_usage(pid)
     for _ = 1, 3 do
       feed_command('so '..fname)
+      wait()
```

Comment:
do we need all these `wait`s ?

Reply:
`wait()` is needed after every `:source` so that the upper/lower tolerance values are the same as in Vim. I added `load_adjust` because the test can still fail on FreeBSD. Without the wait, the upper tolerance must be `time * 2` instead of `load_adjust(time * 1.5)`.

Question is why `:source` returns immediately.